core/lib/spree/testing_support/factories/address_factory.rb

```diff
@@ -11,7 +11,7 @@
     end
 
     firstname 'John'
-    lastname 'Doe'
+    lastname nil
```

Comment:
@gmacdougall @jrochkind any further thoughts on this?  We've been using this in our app for a while now and are happy with it.